docs/functions.json

```diff
@@ -528,20 +547,20 @@
         "detail": "An optional third argument `humanize` can be false to disable the use of thousand separators.",
         "examples": [
             {
-                "template": "@(format_number(31337))",
-                "output": "31,337.00"
+                "template": "@(format_number(1234))",
+                "output": "1,234"
             },
             {
-                "template": "@(format_number(31337, 2))",
-                "output": "31,337.00"
+                "template": "@(format_number(1234.5678))",
+                "output": "1,235"
```

Comment:
Have changed `format_number` so it defaults to zero decimal places instead of 2, so that we don't end up adding .00 to everybody's contact field's for age and other whole numbers. `FIXED(x)` in legacy expressions which defaults to 2 places, migrates to `format_number(x, 2)`.

Reply:
I guess on further thought this could be weird for people with results that actually are decimals.. like currency.. is that common? Maybe it should be adaptive by default?

Reply:
Seems like default should be number of significant places?

Reply:
IE not `0` places. And if you want to force places then you pass in an argument?

Reply:
so basically avoid trailing zeros, right? I think that's reasonable.

Reply:
updated so now..

```
format_number(1234) -> 1234
format_number(1234.000) -> 1234
format_number(1234.500) -> 1234.5
format_number(1234.567) -> 1234.567
```

Reply:
Yep, I think that makes the most sense. When you want fixed then you can pass a parameter for that.